documentation/pwa/tutorial-pwa-icons.asciidoc

```diff
@@ -15,12 +15,12 @@ Vaadin uses and serves default PWA icons automatically, but you can use a custom
 
 To use a custom icon image:
 
-. Create an icon image named `icon.png`. The icon must be in PNG format. 
-. Add the image to your `src/main/webapp/icons/` folder. 
+. Create an icon image named `icon.png`. The icon must be in PNG format.
+. Add the image to `icons/` in your static web resources (`src/main/resources/META-INF/resources/icons/` in Spring projects, `src/main/webapp/icons/` for non-Spring projects).
```

Comment:
I think src/main/resources also work, wonder if it worth mentioning, or maybe it's better than src/main/webapp?